crates/optimism/primitives/src/transaction/signed.rs

```diff
@@ -44,8 +45,14 @@ pub struct OpTransactionSigned {
     #[deref]
     #[as_ref]
     pub transaction: OpTypedTransaction,
+
+    /// Can we attach a conditional the moment a transaction is deserialized?
+    pub conditional: Option<TransactionConditional>
 }
 
+// TEMPORARY since TransactionConditional does not impl eq
```

Comment:
mind submitting a pr on alloy for this?

Reply:
yes definitely

Reply:
Looping back to these changes this week. Saw https://github.com/alloy-rs/alloy/pull/1953. Thanks for adding that in there